tests/test_apetag.cpp

```diff
@@ -89,10 +89,10 @@ class TestAPETag : public CppUnit::TestFixture
   void testPropertyInterface2()
   {
     APE::Tag tag;
-    APE::Item item1 = APE::Item("TRACK", String("17"));
+    auto item1 = APE::Item("TRACK", String("17"));
```

Comment:
wonder if it makes more sense to do

```
APE::Item item1("TRACK", String("17");
```

to look more like APE::Tag tag;

Reply:
OK, I will change this and the other similar items.

Reply:
All except one of these "No need for the second part" concern _signed_ variables, so they could be negative with specially crafted or fuzzed files, and leaving the second part would let such negative values pass, so I would rather leave it as it is. Or am I missing something?

Reply:
AFAIK if statements are booleans. Negative values evaluate to 0.

Reply:
No, only zero is falsy, see [Boolean Conversions](https://en.cppreference.com/w/cpp/language/implicit_conversion#Boolean_conversions) or try it out [online](https://www.w3schools.com/cpp/trycpp.asp?filename=demo_compiler).